.evergreen/run-orchestration.sh

```diff
@@ -102,6 +102,19 @@ echo 'MONGODB_URI: "'$URI'"' > mo-expansion.yml
 echo $URI > $DRIVERS_TOOLS/uri.txt
 echo "Cluster URI: $URI"
 
+if [ -z $MONGO_CRYPT_SHARED_DOWNLOAD_URL ]; then
+  echo "There is no crypt_shared library for distro='$DISTRO' and version='$MONGODB_VERSION'".
+else
+  echo "Downloading crypt_shared package from $MONGO_CRYPT_SHARED_DOWNLOAD_URL"
+  download_and_extract_crypt_shared "$MONGO_CRYPT_SHARED_DOWNLOAD_URL" "$EXTRACT" CRYPT_SHARED_LIB_PATH
+  echo "CRYPT_SHARED_LIB_PATH:" $CRYPT_SHARED_LIB_PATH
+
+cat <<EOT >> mo-expansion.yml
+CRYPT_SHARED_LIB_PATH: "$CRYPT_SHARED_LIB_PATH"
```

Comment:
In .Net EG scripts we had a bug where the shared lib was downloaded, but CRYPT_SHARED_LIB_PATH was not passed through. So we added an explicit validation for that (which is being removed with this change).

Should we validate that CRYPT_SHARED_LIB_PATH is set here (when shared lib is requested)? Or maybe validate that in our .Net scripts? Or not worry about it at all?
I am afraid that due to some future bug/misconfiguration, we might silently fallback to mongocryptd, and that might go unnoticed for quite a while.
cc @kevinAlbs

Reply:
I'm ok to add validation on this level, will do it tomorrow